### ISSUE TITLE
Inconsistency between Lua Editor and Material Editor when attempting to launch an already opened tool

### ISSUE THREAD

**Issue by LB-BartoszCwiek** [o3de/o3de#8733]
**Describe the bug**
Launching the Material Editor when it is already opened causes its Windows task bar icon to flash. This behavior does not occur for the Lua Editor.
Additionally, when launching the Material editor, the Console displays the following text: '(MaterialComponent) - Launching Material Editor'. There is no such information when launching the Lua Editor.
Please refer to the attached video for more details.
This issue also occurs on development branch but it is harder to notice because of the https://github.com/o3de/o3de/issues/7113 issue.

**Steps to reproduce**
Steps to reproduce the behavior

1. Go to Tools > Lua Editor.
2. Without closing the Lua Editor repeat step 1.
3. Repeat steps 1-2 using the Material Editor.

**Expected behavior**
There is no inconsistency between Lua Editor and the Material Editor when it comes to the Console output and task bar icon.

**Actual behavior**
There is an inconsistency between Lua Editor and the Material Editor when it comes to the Console output and task bar icon.

**Video**

https://user-images.githubusercontent.com/100559077/161949706-499b4f76-0f61-4395-a9ba-a120f9ebf55c.mp4

**Found in Branch**
Stabilization (3e47df2)
Development (63aadae)

Device: PC
OS: Windows
Version: 10
CPU: Intel Core i7-11700F
GPU: NVIDIA GeForce RTX 3070
Memory: 32GB

**Comment by gadams3** [comment]
After reviewing the video multiple times and trying to reproduce the issue locally, it appears that both applications are behaving exactly the same when reopening them except that material editor prints the message in the output window. In the video, the user minimize the material editor but did not minimize the lua editor. If either application is minimized and you attempt to reopen them from the main editor menu or related components then the icons will flash on the taskbar but the windows will not be shown. If the windows were not minimized then they will become active and visible immediately when requested from the main editor. This is the behavior on windows OS.

In summary, the reporter did not follow the same steps for both applications. If either application is minimized then attempting to reopen it from the main editor will only flash on be taskbar to notify the user to take action.
